tests/queries_/models.py

```diff
@@ -53,3 +59,16 @@ class Meta:
 
     def __str__(self):
         return str(self.pk)
+
+
+class Writer(EmbeddedModel):
+    name = models.CharField(max_length=10)
+
+
+class Article(models.Model):
+    headline = models.CharField(max_length=100)
+    number = models.IntegerField()
+    body = models.TextField()
+    location = models.JSONField(null=True)
```

Comment:
NIT: (Fun fact) This could also have been an EmbeddedModelField for a `location` object. 
```
class Location(EmbeddedModel):
    type = models.CharField(default="Point")
    coordinates = models.ArrayField(FloatField(), max_size=2)
```